example/mouse_296929.json

```diff
@@ -1,8 +1,10 @@
 {
+    "version": 0.1,
```

Comment:
Confirming that this version shows up in the mouse schema.

Reply:
I haven't found the way to actually get versions into schemas. I've found information on the value of versioning, but not how to best do it - which is why the files are out of sync.
Also, the examples are not up to date with the current schemas as I've been working on them in fits and bursts. I'll clean them up when I'm back in the office next week.

Reply:
It's possible to add it as a constant in the schema. Then any instance will invalidate if it doesn't match the constant. So in the `Mouse Schema.json` file it could be set as:

```
"properties": {
		"version": {
			"type": "string",
			"const": "0.0.1",
			"description": "schema version"
		},
```